plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -395,6 +395,24 @@ public void testUnregisterTableAccessControl()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testCreateTableInNonExistentSchemaFails()
+    {
+        String tableName = "test_create_table_in_non_existent_schema_" + randomNameSuffix();
+        String location = generateTableLocation(tableName);
+        assertQueryFails(
+                "CREATE TABLE non_existing_schema." + tableName + " (a int, b int) WITH (location = '" + location + "')",
+                "Schema non_existing_schema not found");
+        assertThat(locationExists(location))
```

Comment:
I suppose the test purpose is verifying the location wasn't created. It would be better to rename the test method. Otherwise, it's duplicate of `BaseConnectorTest#testCreateTableSchemaNotFound`.

Reply:
Modified test name to `testCreateTableWithNonExistingSchemaVerifyLocation`